packages/terra-table/src/TableCell.jsx

```diff
@@ -1,23 +1,38 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
 import 'terra-base/lib/baseStyles';
+import styles from './TableCell.module.scss';
+
+const cx = classNames.bind(styles);
 
 const propTypes = {
   /**
    * Content to be displayed for the row cell
    */
-  // eslint-disable-next-line react/forbid-prop-types
-  content: PropTypes.any.isRequired,
+  children: PropTypes.node,
```

Comment:
Based on conversation below, do we need default props here for children and refCallback? https://github.com/cerner/terra-core/pull/2159/files#r248783250

Reply:
Updated.